src/pages/fr/core-concepts/framework-components.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Composants Framework
+description: Apprennez à utiliser React, Svelte, etc.. avec Astro
+---
+
+Construisez votre site Astro sans sacrifier votre Framework favori.
+
+Astro supporte un large choix de Frameworks comme [React](https://reactjs.org/), [Preact](https://preactjs.com/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/), [SolidJS](https://www.solidjs.com/), [AlpineJS](https://alpinejs.dev/) et [Lit](https://lit.dev/).
+
+## Installation d'intégrations
+
+Astro peut être installé avec des intégrations pour React, Preact, Svelte, Vue, SolidJS et Lit. Une ou plusieurs de ces intégrations peuvent être installées et configurées dans votre projet.
+
+Afin de configurer Astro pour utiliser ces Frameworks, d'abord, installez son intégration et toutes ses dépendances associées :
+
+```bash
+npm install --save-dev @astrojs/react react react-dom
+```
+
+Ensuite, importez et ajoutez la fonction à votre liste d'intégrations dans `astro.config.mjs` :
+
+```js
+import { defineConfig } from 'astro/config';
+
+import react from '@astrojs/react';
+import preact from '@astrojs/preact';
+import svelte from '@astrojs/svelte';
+import vue from '@astrojs/vue';
+import solid from '@astrojs/solid-js';
+import lit from '@astrojs/lit';
+
+export default defineConfig({
+	integrations: [react(), preact(),svelte(), vue(), solid() , lit()],
+});
+```
+
+⚙️ Consultez le [Guide d'Intégrations](/fr/guides/integrations-guide/) pour plus de détails sur l'installation et la configuration d'intégrations Astro.
+
+⚙️ Vous voulez voir un exemple pour le Framework de votre choix ? Visitez [astro.new](https://astro.new/) et sélectionnez un des modèles de Framework disponible.
+
+## Utilisation des composants de Framework
+
+Utilisez vos composants de Framework JavaScript dans vos composants Astro, Pages et Layouts comme des composants Astro classiques ! Tous vos composants peuvent être placés dans `/src/components`, ou peuvent être organisés de la manière que vous le souhaitez.
+
+Pour utiliser un composant de Framework, importez-le à partir de son chemin relatif (y compris l'extension de fichier) dans le script du composant Astro. Ensuite, utilisez le composant avec d'autres composants, des éléments HTML et des expressions similaire au JSX dans le modèle du composant.
+
+```astro
+---
+import MyReactComponent from '../components/MyReactComponent.jsx';
+---
+<html>
+  <body>
+    <h1>Utilisez des composants React directement dans Astro !</h1>
+    <MyReactComponent />
+  </body>
+</html>
+```
+
+> 💡 _N'oubliez pas : Toutes les importations doivent être **en haut** de votre script de composant Astro._
+
+Par défaut, vos composants de Framework seront rendus en HTML statique. C'est utile pour les composants qui ne sont pas interactifs et qui évite de transmettre à l'utilisateur du JavaScript inutile.
```

Comment:
Maybe remove "qui" from "...et qui évite de transmettre"? Then maybe "C'est utile... et cela évite...". I needed to read a few times before concluding that "qui" didn't refer to "les composants".